### DISCUSSION TITLE
稀土掘金上面的文章是获授权的吗？

### DISCUSSION THREAD

**Discussion by LinusLieu** [lss233/chatgpt-mirai-qq-bot#276] [General]
单纯看到了内容一毛一样的文章，过来提醒一下
[https://juejin.cn/post/7206219564090130487](url)

**Comment by lss233** [comment]
你在哪里看到了一模一样的内容呢？

**Reply by LinusLieu** [reply]
https://juejin.cn/post/7206219564090130487

就上面这个链接

**Reply by HBLADEH** [reply]
这个应该是群友写的！？
